src/DependencyInjection/JsonRpcMethodDefinitionHelper.php

```diff
@@ -11,6 +12,22 @@
  */
 class JsonRpcMethodDefinitionHelper
 {
+
+    private $annotationsEnabled;
+
+    private $reader;
+
+    private const METHOD_ANNOTATION_CLASS = 'Yoanm\\SymfonyJsonRpcHttpServer\\Annotation\\JsonRpcMethod';
+
+    public function __construct(ContainerBuilder $container)
+    {
+        $this->annotationsEnabled = $container->has('routing.loader.annotation')
+            && class_exists('Doctrine\Common\Annotations\AnnotationReader');
+        if ($this->annotationsEnabled) {
+            $this->reader = new \Doctrine\Common\Annotations\AnnotationReader();
```

Comment:
```suggestion
            $this->reader = new AnnotationReader();
```
Use a `use` statement instead. Nothing will be broken even if class doesn't exist

Reply:
Good point - still needs to be a string in the `class_exists()` invocation but TIL `use` is just a lexer replacement.